lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -987,6 +1010,7 @@ class SqlParser(private val ion: IonSystem) : Parser {
             "substring" -> tail.parseSubstring(head!!)
             "trim" -> tail.parseTrim(head!!)
             "extract" -> tail.parseExtract(head!!)
+            "let" -> tail.parseLet()
```

Comment:
I just bumped into this by accident--this needs to be removed.  It's not helping anything since there are not test cases that cover this.  This might be left over from when we considering allowing `LET` to be a first-class expression.  In any case, this should be removed here...

Reply:
Will remove when merging `from-let-final` into `master`.

Reply:
Removed in this [commit](https://github.com/partiql/partiql-lang-kotlin/pull/303/commits/07b654aca2c75e4fb57dce77712b93a9509b7d6e).